include/onnxruntime/core/session/onnxruntime_c_api.h

```diff
@@ -1252,7 +1276,7 @@ struct OrtApi {
   /// \name OrtMemoryInfo
   /// @{
 
-  /** \brief Create an OrtMemoryInfo
+  /** \brief Create an ::OrtMemoryInfo
```

Comment:
C programmers are not supposed to know what "::" means.

Reply:
It's to make the symbols show up in Doxygen as clickable links, otherwise it's just text.